lib/galaxy_test/api/test_workflows.py

```diff
@@ -3082,15 +3082,6 @@ def test_workflow_invocation_report_custom(self):
             assert "## Workflow Inputs" in markdown_content
             assert "## About This Report" in markdown_content
 
-    @skip_without_tool("cat1")
-    def test_export_invocation_bco(self):
```

Comment:
We should probably keep the test but update `get_biocompute_object` function to use the short-term download API.

I guess calling `self.workflow_populator.download_invocation_to_store(invocation_id, extension="bco.json")` will do the trick.

https://github.com/galaxyproject/galaxy/blob/a5d66888c9ca37f821b8510ca4c4a5852ba2ff7d/lib/galaxy_test/base/populators.py#L1790

Reply:
I saw this test but having an API test is surely a good idea, added a commit, thanks for the review!

https://github.com/galaxyproject/galaxy/blob/dev/test/integration/test_workflow_tasks.py#L66-L70